share/config_global.toml

```diff
@@ -169,3 +169,18 @@ name = "Other"
         choices.ipv4 = "IPv4 Only"
         choices.ipv6 = "IPv6 Only"
         default = "both"
+
+    [misc.sni_forward]
+    name = "SNI-based forwarding"
+
+        [misc.sni_forward.sni_forward_enabled]
+        type = "boolean"
+        default = false
+
+        [misc.sni_forward.sni_forward_list]
+        type = "tags"
+        # Regex is just <domain regex>:<ip regex>
+        pattern.regexp = '^([^\W_A-Z]+([-]*[^\W_A-Z]+)*\.)+((xn--)?[^\W_]{2,}):((25[0-5]|(2[0-4]|1\d|[1-9]|)\d)\.?\b){4}$'
```

Comment:
What the problem to accept IPv6 too ?

Reply:
We also could imagine someone who rent IPv6 VPS (some provider do it for free) and wants to assemble all the thing on a VPS with IPV4

Reply:
Marking as resolved because this is an advanced use case and let's focus on nominal / classic use case first